scripts/experimental/install_dev_osgeo.sh

```diff
@@ -131,6 +131,11 @@ else
     GDAL_DL_URL="https://download.osgeo.org/gdal/${GDAL_VERSION}/gdal-${GDAL_VERSION}.tar.gz"
 fi
 
+## ensure that osgeo.gdal and family get installed for python use https://github.com/OSGeo/gdal/blob/master/swig/python/README.rst
+wget https://bootstrap.pypa.io/get-pip.py
+python3 get-pip.py
+pip install numpy setuptools
```

Comment:
I don't see the need for this, but could you at least change the install command to not leave the cache in the layer and delete what wget downloaded?

https://github.com/rocker-org/rocker-versioned2/blob/5fa99b369510466ea8a2803d29e3417f5212a0fc/scripts/install_jupyter.sh#L32

Reply:
yes sorry, bit out of my depth I think, I'll keep exploring this and the other python pathway

the need is to have osgeo.gdal available on the same dev-osgeo basis, and callable from R

Reply:
I understand what the process you wrote is trying to do. For consistency's sake, I'd like pip not to be installed this way.
How about using the `install_python.sh` script?

Reply:
oh thanks yes! will do , never before understood what *should* be done because the official-seeming docs only show ensure-pip and this weird script way